new-backoffice/extending/development-flow/vite-package-setup.md

```diff
@@ -126,8 +131,27 @@ This example declares a Dashboard as part of your Package, using the Vite exampl
 
 {% endcode %}
 
+{% hint style="info" %}
+Umbraco needs the name of the element that will render as default when our dashboard loads. This is specified in the manifest as the `elementName`. Another approach would be to define your default element in the TS code.
+
+To do this, you should add `default` to your `MyElement` class in the `my-element.js` file like so 
+
+```ts
+export default class MyElement extends LitElement {
+```
+
+{% endhint %}
+
+{% hint style="info" %}
+
+For this to work on your Umbraco website, you need to include all the files in the `src` folder and the `umbraco-package.json` file. We recommend you do this by adding a few lines to your csproj file like so:
+
+If you try to include these resources via Visual Studio, be careful to include only the `dist` folder. Otherwise, VS will try to be smart by including a few lines on your csproj file to compile the TypeScript code that exists in your project folder. When you run your website, VS will try to compile these files and will fail.
+
+{% endhint %}
```

Comment:
there are 2 info notes one after another. To avoid this, my suggestion would be for the 2nd info note to be changed to a new headline like "#### Including files" or just leave it as normal text or add in the previous info note (which in this case the first info note might be longer). Another idea would be to change the 2nd info note to warning as it mentions "be careful to include only the `dist` folder..."